### PR TITLE
Changes to add krun to the kontain release

### PR DESCRIPTION
**Pull request by paulpopelka**
Update getting started doc to describe how to use krun.
Change the install script to verify that packages krun depends upon are installed.

### CODE REVIEW

File: GettingStarted.md [hunk 1]
```diff
@@ -131,28 +131,59 @@ You can run Kontain payload wrapped in a native Docker container.... in this cas
...
+* krun package requirements
+
+krun needs the following packages to run:
+
+glibc
+yajl
+libseccomp
+libcap
+
+Install them with the appropriate package manager for your linux distribution.  Probably dnf or apt-get.
```
Comment (msterin) [inline GettingStarted.md:185]: pls add specific example for at least dnf

**Review by msterin — APPROVED** [review]
LGTM with minor nits. 
One request - please file an issue for adding examples from the GettingStarted to CI pass on the release. Just create it, no need to assign or triage

File: kontain-install.sh [hunk 2]
```diff
@@ -26,6 +27,17 @@ function error {
    exit 1
 }
 
+pkgs_missing=0
+function check_packages {
+    echo "Checking that packages $NEEDED_PACKAGES are present"
+    for i in $NEEDED_PACKAGES ; do
+        if ! rpm -qa | grep $i -q ; then
+            pkgs_missing=1
+            warning "Package $i is required and is not installed."
```
Comment (msterin) [inline kontain-install.sh:36]: Pls add something like 'docker runtime `krun` may not function without it`

File: GettingStarted.md (hunk 1 above)
Comment (paulpopelka) [reply GettingStarted.md]: Done.

**Pull request merged**